Src/MoneyFox/ViewModels/Payments/PaymentListViewModel.cs

```diff
@@ -146,7 +146,7 @@ private void CalculateSubBalances(DateListGroupCollection<PaymentViewModel> grou
         {
             group.Subtitle = string.Format(
                 format: Strings.ExpenseAndIncomeTemplate,
-                arg0: group.Where(x => x.Type == PaymentType.Expense || x.Type == PaymentType.Transfer && x.ChargedAccount.Id == SelectedAccount.Id)
+                arg0: group.Where(x => (x.Type == PaymentType.Expense || x.Type == PaymentType.Transfer) && x.ChargedAccount.Id == SelectedAccount.Id)
```

Comment:
I think this could be rewritten to be more explicit with testing on
 ```
x.Type is not PaymentType.Income && x.ChargedAccount.Id == SelectedAccount.Id